aws/resource_aws_elb_test.go

```diff
@@ -1952,7 +1952,7 @@ resource "aws_subnet" "public_a_two" {
 }
 
 resource "aws_elb" "test" {
-  name = "terraform-asg-deployment-example"
+  name = "terraform-asg-deployment-2-1"
```

Comment:
Since [`name` is a `ForceNew` attribute](https://github.com/terraform-providers/terraform-provider-aws/blob/cd9ca85fbca8e7dc2be953231b435fd29f1b1394/aws/resource_aws_elb.go#L35-L42), we do not want the resource to be recreated between test steps.

```suggestion
  name = "terraform-asg-deployment-example"
```